reinforcement_learning/requirements.txt

```diff
@@ -0,0 +1,4 @@
+tensorflow-gpu==1.10
```

Comment:
nit: Have you guys tested this in TF 1.12 ? CMLE will be releasing TF 1.12 end of this week.

Reply:
No. Thanks for the info, I'll test it once 1.12 is available.
For this reason, can I postpone the modification of README.md after that? Since some results may change and I'd like to update gifs.

Reply:
Yes, that's fine, just a comment, let's keep it in TF 1.10 then and make sure all your code is reviewed and submitted. Later you can test it TF 1.12 and update if needed.